debug_module.tex

```diff
@@ -110,8 +110,10 @@ \section{Selecting Harts} \label{selectingharts}
 
 \section{Run Control} \label{runcontrol}
 
-The Debug Module can halt selected harts and allow them to run again using
-the \Rdmcontrol register.
+For every hart, the Debug Module contains 3 bits: halt request, resume request,
+and hart reset. (The hart reset bit is optional.) These bits all reset to 0. A
```

Comment:
they only reset to 0 when debug logic is reset (maybe this is obvious, maybe not)

Reply:
Obvious to me. Same reset as is mentioned in every register description. I don't think it's necessary to list what signal does the reset, since our descriptions of the reset signals already list what they reset.